_posts/2018-07-05-client-provisioning-methods.adoc

```diff
@@ -6,45 +6,40 @@
 
-* *Automatic provisioning*
+* *Provisioning with shared credentials*
 
 Devices use temporary provisioning credentials to request permanent credentials from the server. You download the temporary provisioning credentials from the OTA Connect web app and install them on a device. Once the device registers with the server for the first time, the server assigns permanent credentials to the device. The device then uses these permanent credentials for all future transactions.
 
 
-* *Implicit Provisioning*
+* *Provisioning with device credentials*
 +
 Devices already have permanent credentials installed. The server doesn't issue any credentials to devices. Instead, you use a root CA certificate to sign the credentials that you install on the device. You then install the same root CA certificate on the OTA Connect server.
 Every time the device attempts to connect, the server verifies that the device credentials are signed by the same CA that you originally installed on the server.
 
 == Choosing a provisioning method
 
-The type of provisioning that you choose depends on your requirements. If you're just testing and want to get started quickly, automatic provisioning is fine. It is easier to set up but is less secure. Because server self-signs the credentials that it issues to devices, the devices have no way of verifying the integrity of the server. This level of security is not suitable for a production environment. The device should always be able to verify that is communicating with a genuine OTA Connect server. Once you move to production, you should look at some form of implicit provisioning. The most common way to implicity provision devices is with a Hardware Security Module (HSM).
+The type of provisioning that you choose depends on your requirements. If you're just testing and want to get started quickly, provisioning with shared credentials is fine. It is easier to set up but is less secure. Because server self-signs the credentials that it issues to devices, the devices have no way of verifying the integrity of the server. This level of security is not suitable for a production environment. The device should always be able to verify that is communicating with a genuine OTA Connect server. Once you move to production, you should look at some form of provisioning that uses device certificates. For more security, use a Hardware Security Module (HSM) to store the device certificates.
 
-=== Enable automatic provisioning
+=== Configure a device to provision with shared credentials
 
-To enable automatic provisioning, follow these steps:
+To configure a device to provision with shared credentials, follow these steps:
 
 . Download a link:../quickstarts/generating-provisioning-credentials.html[provisioning key].
 . Edit the following `.toml` configuration file:
 +
 link:https://github.com/advancedtelematic/aktualizr/blob/master/config/sota_autoprov.toml[config/sota_autoprov.toml]
```

Comment:
@patrickvacek, let's try to update the configure file as well and get rid of '_auto_'

Reply:
Yes, of course. That has to be done in the aktualizr repo and then we can come back to this and clean it up.